Northstar.CustomServers/mod/scripts/vscripts/burnmeter/_burnmeter.gnut

```diff
@@ -494,44 +495,61 @@ void function PlayerUsesPhaseRewindBurncardThreaded( entity player )
 	for ( int i = positions.len() - 1; i > -1; i-- )
 	{
 		// should looks better?
+		player.UnforceCrouch()
+		player.UnforceStand()
+		if( positions[ i ].wasCrouched )
+			player.ForceCrouch() // make player's view lower
+		else
+			player.ForceStand()
 		float moveTime = PHASE_REWIND_PATH_SNAPSHOT_INTERVAL + 0.1
 		player.SetVelocity( -positions[ i ].velocity )
 		mover.NonPhysicsMoveTo( positions[ i ].origin, moveTime, 0, 0 )
 		mover.NonPhysicsRotateTo( positions[ i ].angles, moveTime, 0, 0 )
 		wait PHASE_REWIND_PATH_SNAPSHOT_INTERVAL
 	}
 	player.SetVelocity( <0, 0, 0> )
+	player.UnforceCrouch() // clean up!
+	player.UnforceStand()
 	if( positions[0].wasCrouched )
-		player.SetOrigin( player.GetOrigin() + < 0, 0, 20 > )
+		thread PhaseRewindForceCrouch( player )
+	player.SetOrigin( positions[0].origin + < 0,0,20 > )

```

Comment:
```suggestion
	player.SetOrigin( positions[0].origin + < 0, 0, 20 > )

```

Reply:
I changed this after my own testing, it may works better now